README.md

```diff
@@ -293,6 +294,17 @@ kustomize build contrib/kserve/models-web-app/overlays/kubeflow | kubectl apply
 
 - ../contrib/kserve/models-web-app/overlays/kubeflow
 
+
+#### BentoML
```

Comment:
Also remove this section as we don't list contrib components in the readme

Reply:
Removed